backend/api/Areas/Property/Mapping/Building/BuildingMap.cs

```diff
@@ -65,6 +95,12 @@ public void Register(TypeAdapterConfig config)
                 .Map(dest => dest.IsSensitive, src => src.IsSensitive)
                 .Map(dest => dest.Evaluations, src => src.Evaluations)
                 .Map(dest => dest.Fiscals, src => src.Fiscals)
+                .AfterMapping((src, dest) =>
+                {
+                    var metadata = JsonSerializer.Serialize<IEnumerable<Model.LeasedLandMetadataModel>>(src.LeasedLandMetadata, _serializerOptions);
```

Comment:
Might want to also do the check for null `src.LeasedLandMetadata ?? "{}"`

Reply:
Yeah, I have a fix for this on my local.